.github/workflows/binder.yaml

```
@@ -10,3 +10,5 @@ jobs:
       pull-requests: write
     steps:
       - uses: jupyterlab/maintainer-tools/.github/actions/binder-link@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Maybe we can try with `github-token`:

```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

---

Hmm although this doesn't seem to be what the action expects here:

https://github.com/jupyterlab/maintainer-tools/blob/4af624a8519f8887712c7932931546399b0ff14d/.github/actions/binder-link/action.yml#L4

---

Maybe lowercase `GITHUB_TOKEN`:

```suggestion
          github_token: ${{ secrets.github_token }}
```

---

My first attempt which failed was with `github-token`, that why I changed it in a new commit.
The `maintainer-tools` seems to wait for `github_token` input, and to rename it `github-token`  : 
https://github.com/jupyterlab/maintainer-tools/blob/4af624a8519f8887712c7932931546399b0ff14d/.github/actions/binder-link/action.yml#L16
I'll try to run again the failed workflow.